cookbooks/clearwater/templates/default/shared_config.erb

```diff
@@ -52,9 +52,9 @@ trusted_peers="<%= node[:clearwater][:trusted_peers].join "," %>"
 <% if @node[:clearwater][:scscf] %>scscf=<%= @node[:clearwater][:scscf] %><% end %>
```

Comment:
Worth a comment (http://stackoverflow.com/questions/3426671/how-does-one-comment-in-an-erb-template) to say that we use @node[:clearwater][:icscf] to make these overrideable in the environment?